src/features/settings/pages/PracticePage.tsx

```
@@ -418,205 +381,207 @@ export const PracticePage = ({ className, onBack }: PracticePageProps) => {
-                />
+              <div className="mt-0.5 font-mono text-xs text-dim">
+                {contactValues.businessEmail || currentPractice.businessEmail || ''}
               </div>
             </div>
           </div>
-        </SettingSection>
 
-        <SectionDivider />
+          <div className="grid grid-cols-2 gap-[14px] max-[720px]:grid-cols-1">
+            <div className="form-field">
+              <label className="label">Firm name</label>
```

ESLint

[error] A form label must be associated with a control.

(jsx-a11y/label-has-associated-control)